lua/dashboard/theme/hyper.lua

```diff
@@ -112,7 +112,8 @@ local function project_list(config, callback)
       else
         reverse(res)
       end
-      table.insert(res, 1, config.project.icon .. ' Recently Projects: ')
+      table.insert(res, 1, '')
```

Comment:
there just for packages.enable = false right ?

Reply:
no, but visuals are not changed if packages remains enabled.
empty line is moved from bottom of lines {} to top of res{}. 
This way, regardless of packages being enabled, there is one line of padding between Recent Projects and shortcuts / packages

Reply:
I had add it and do some change . you can test it.